ui-admin/src/help/ExportHelp.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+
+/** guide to using the participant export/download function */
+export default function ExportHelp() {
+  return <div>
+    <h3>Participant List Export Info</h3>
+    <p>Participant export enables download of tabular files (.tsv or .xlsx) containing all participants
+      except those who have withdrawn from the study.
+            One row will be generated per participant.</p>
+
+    <h4>File format</h4>
+    <div className="panel">
+      <ul>
+        <li>
+          <b>.xlsx</b> will create an Excel spreadsheet of the participant data. Empty cells will represent
+                    null values.
+        </li>
+        <li>
+          <b>.tsv</b> (tab-delimited values). Will export a tab-delimited file. This may be useful in
+                    environments where Excel is unavailable, or if the
+                    number of columns to be exported exceeds 16K. In order to have data be compliant, double-quotes will
+                    be replaced by single quotes, and any values including
```

Comment:
It isn't really necessary to replace double quotes with single quotes. It's unlikely that the meaning of any responses will be different, but I have to confess to being a little uneasy about any modification of responses.

Reply:
you're right that they can be escaped, but other applications handling of escaped quotes in csv files varies enough that it was decided for Pepper that this strategy would result in less user issues (and they can always grab the .xlsx format if quote fidelity is important)